src/index.js

```diff
@@ -4,7 +4,7 @@ import sizeMe from 'react-sizeme';
 sizeMe.noPlaceholders = true;
 
 // Styles
-export styles from './styles.scss';
+// export styles from './styles.scss';
```

Comment:
```suggestion

```

Reply:
this was removed in the refactor and we are not compiling styles by using sass entry point.